analyses/molecular-subtyping-LGAT/04-LGAT-compile-subtypes.Rmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "Compile LGAT subtyping"
+output: html_notebook
+---
+
+In this notebook, we will be compiling the subtype annotation for LGAT samples. Subtypes are described in [#790](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/790) which were gathered through the following scripts : 
+
+`01-subset-files-for-LGAT.Rmd` gathers subtype associated SNV annotation 
+`02-subset-fusion-files-LGAT.Rmd` gathers subttype associated Fusion annotation
+`03-subset-cnv-files-LGAT.Rmd` gathers subtype associated CNV annotation
+
+along with the above molecular data based annotation in this notebook we will include NF-1 cancer_predispositions from histology file as a criteria for LGG, NF1 as well
+
+### Setup
+```{r}
+library("tidyverse")
+
+# get subset folder
+subset_dir <- file.path(
+  root_dir,
+  "analyses",
+  "molecular-subtyping-LGAT",
+  "lgat-subset"
+)
+
+# result Dir
+resultDir <- file.path(
+  root_dir,
+  "analyses",
+  "molecular-subtyping-LGAT",
+  "results"
+)
+
+```
+
+### Input
+```{r}
+# Look for git root folder
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# File from 00-LGAT-select-pathology-dx that is used for the pathology diagnosis
+# inclusion/exclusion criteria
+path_dx_list <- jsonlite::fromJSON(
+  file.path(subset_dir, 
+            "lgat_subtyping_path_dx_strings.json")
+)
+
+# clinical file
+lgat_manifest <- read_tsv(file.path(root_dir, 
+                                    "data",
+                                    "pbta-histologies-base.tsv"), 
+                          guess_max = 10000) %>%
+  # use inclusion/exclusion criteria from pathology_diagnosis
+  dplyr::filter(str_detect(str_to_lower(pathology_diagnosis),  # Inclusion criteria
+                           paste(path_dx_list$include_path_dx, collapse = "|")),
+                # Exclusion criteria
+                str_detect(str_to_lower(pathology_diagnosis),
+                           paste(path_dx_list$exclude_path_dx, collapse = "|"),
+                           negate = TRUE),
+                # Tumors
+                sample_type == "Tumor",
+                composition == "Solid Tissue") %>%
+  # select columns for interest
+  select(Kids_First_Biospecimen_ID,
+         Kids_First_Participant_ID,
+         sample_id,
+         experimental_strategy,
+         cancer_predispositions)
+
+lgat_wgs <- lgat_manifest %>%
```

Comment:
I take it that no WXS samples are included after the inclusion/exclusion criteria?

Reply:
right, I mean I know that LGAT doesn't have any WXS. I can make it more generic `lgat_dna`